.changeset-staged/eight-impalas-return.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/core": minor
+"@logto/connector-kit": minor
+---
+
+The `getUserInfo` method of connectors has been passed the `storage.get` and `storage.set` methods so that the connector instance can store and retrieve access token related information in the database.
```

Comment:
```suggestion
The `getUserInfo` method of connectors has been passed to the `storage.get` and `storage.set` methods so that the connector instance can store and retrieve access token related information in the database.
```